ui/src/app/auth/auth.guard.ts

```diff
@@ -1,6 +1,6 @@
 import {Injectable} from "@angular/core"
 import {ActivatedRouteSnapshot, CanActivate, Router, RouterStateSnapshot} from "@angular/router"
-import {ENABLE_ROLES} from "./auth-roles"
+import {ENABLE_ROLES, ACTION_ROLES} from "./auth-roles"
```

Comment:
Are we using the ACTION_ROLES import here?

Reply:
Removed unused import.